.mage/magefile.go

```diff
@@ -25,15 +27,34 @@ import (
 	"go.einride.tech/mage-tools/targets/mggoreview"
 )
 
+func init() {
+	mgmake.AddExtraMakefiles(
```

Comment:
As discussed, let's call it `GenerateMakefiles` to make it more self-describing what the method does